### DISCUSSION TITLE
Annotation opacity as an option

### DISCUSSION THREAD

**Discussion by codyseally** [sumatrapdfreader/sumatrapdf#3888] [Ideas]
Currently in Advanced Options, for annotations, we have: 

```
Annotations [
	HighlightColor = #ffff00
	UnderlineColor = #00ff00
	SquigglyColor = #ff00ff
	StrikeOutColor = #ff0000
	FreeTextColor = 
	FreeTextSize = 12
	FreeTextBorderWidth = 1
	TextIconColor = 
	TextIconType = 
	DefaultAuthor = 
]
```

Would it be possible to add Opacity as well? I find the Opacity = 255 (when you edit every single annotation) to be a bit annoying on the eyes.

**Comment by GitHubRulesOK** [comment]
try adding to the colour some should accept such values here 80/FF=128/256
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/e72edf4f-878d-478e-a01e-29c59170750e)
